data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-consensual-creation-v1.0.xml

```diff
@@ -6,7 +6,7 @@
     <changeSet author="R3.Corda" id="ledger-consensual-creation-v1.0">
 
         <createTable tableName="consensual_transaction">
-            <column name="id" type="VARCHAR(64)">
+            <column name="id" type="VARCHAR(160)">
                 <constraints nullable="false" primaryKey="true"/>
```

Comment:
Why did this need to change, I don't know the length of our hashes but surely they're constant so shouldn't need to change if we have something working?

Reply:
@mnesbit  pointed out that some of the hash algos we support would not fit into varchar(64) such as SHA-512